.github/workflows/push-docker-container.yml

```
@@ -0,0 +1,57 @@
+name: "Push Docker Container"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Additional tag to push'
+        required: false
+  push:
+    branches:
+      - 'main'
+    tags:
+      - '*'
```

I would recommend to push only certain tags, for example:
```suggestion
      - 'RC*'
      - 'preview*
```

---

During development i absolutely agree, though wouldn't we want real version numbers such as `1.0.0` or `2024.2` to have their image created as well?